src/server/engine_shard_set.cc

```diff
@@ -738,7 +738,8 @@ auto EngineShard::AnalyzeTxQueue() -> TxQueueInfo {
         DbTable* table = db_slice().GetDBTable(trx->GetDbIndex());
         bool can_run = true;
 
-        if (!trx->IsMulti()) {
+        if (trx->IsMulti()) {
```

Comment:
why do you need the empty block?

Reply:
I assume he forgo to insert the code that handles `IterateMultiLocks`

Reply:
I assume he forgot to insert the code that handles `IterateMultiLocks`

Reply:
I have not sent this PR for review :)